rcl_action/QUALITY_DECLARATION.md

```diff
@@ -88,15 +88,29 @@ The results of the test can be found [here](https://ci.ros2.org/view/nightly/job
 
 `rcl_action` has feature tests, which test for proper node state transitions.
 The tests are located in the [test](test) subdirectory.
+New features are required to have tests before being added.
+Currently nightly test results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/rcl_action/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rcl_action/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/rcl_action/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/rcl_action/)
 
 ### Public API Testing [4.ii]
 
-Much of the API in `rcl_action` is tested.
-The tests are located in the [test](test) subdirectory.
+Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [94%](https://ci.ros2.org/job/ci_linux_coverage/lastBuild/cobertura/src_ros2_rcl_rcl_action_src_rcl_action/).
```

Comment:
Same comment here and below about the exact number. Maybe we should say >X%, but not yet achieved the 95% ROS2 developer guide metric.
```suggestion
Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [94%](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rcl_rcl_action_src_rcl_action/).
```